### ISSUE TITLE
Screen gets stucked when using on iOS version above 10 & below 11

### ISSUE THREAD

**Issue by siddharthsuneel** [PageMenu/PageMenu#473]
It happens so frequently page menu gets stucked i.e does not swipes on iOS version 10.1,10.2 & 10.3.

**Comment by aadilHaffez** [comment]
@siddharthsuneel Did you get any solution to that issue?

**Comment by JunaidVizz** [comment]
yeah, I'm also facing this issue the page menu doesn't shit to nxt option some time index changed but the indicator stuck on the index where it was...
have you find any solution ?

**Comment by siddharthsuneel** [comment]
@JunaidVizz  No I haven't found any solution yet. 

@aadilHaffez  What about you, is it working fine on your side ?

In case any of you finds the solution, please update here also. So that it helps everyone

**Comment by JunaidVizz** [comment]
@siddharthsuneel  actually I didn't find the exact place yet where the problem is sometimes it stuck on tap on notification sometime it stuck when the app comes from background and sometimes it just goes on normally for days :(

**Comment by iforaa** [comment]
For all who are still struggling, the problem is in the CAPSPageMenu.swift file. This approach doesn't work on the newer models: 
`currentOrientationIsPortrait = UIDevice.current.orientation.isPortrait`
Replace it with this:
```
currentOrientationIsPortrait = UIDevice.current.orientation.isValidInterfaceOrientation
            ? UIDevice.current.orientation.isPortrait
            : UIApplication.shared.statusBarOrientation.isPortrait
```

**Comment by djerick1989** [comment]
Thanks @iforaa it´s works fine.

**Comment by redroostertech** [comment]
THANKS @iforaa !!!
